src/Modules/Registration/RegistrationContainer.js

```diff
@@ -96,6 +96,10 @@ const RegistrationContainer = (props) => {
     return location.state? `Your reservation time is at ${location.state.event_slot.start_time} - ${location.state.event_slot.end_time} on ${location.state.event_date}. `: "";
   }
 
+  const getCodeURL = (identification_code) => {
+    return `Your QRCode for the Reservation http://localhost:5000/qrcode/${identification_code}  `;
```

Comment:
above URL needs to be updated, to work in  production/beta

Reply:
fixed in commit - https://github.com/midohiofoodbank/freshtrak-client/pull/232/commits/f21ee4ea6f67f760190fa5b24cd11b6c4b982f4d

Reply:
thank you

Reply:
we should also add qrcode in the url

Reply:
@sarath-yerra updated

Reply:
@maheshkurma when you have a chance can you please review this PR.

Reply:
lets fix `qr_code` issue which i see in master branch, before we merge any qr related code.
@sarath-yerra  issue is seen in below screenshot : 
<img width="1440" alt="Screen Shot 2021-01-15 at 9 24 12 AM" src="https://user-images.githubusercontent.com/20909776/104738867-0fbdee00-5714-11eb-94e9-90c1b898d3cd.png">

Reply:
This is fixed and merged to master